map.js

```diff
@@ -447,16 +392,19 @@ function onMouseClick(mouseLoc){
 }
 
 function initListeners(){
-    const CLICK_LIMIT_PIXELS = 5;
+    const CLICK_LIMIT_PIXELS = 8;
```

Comment:
This is so that if the user slightly moves their mouse during a click, it still registers as a click. But if they're dragging quickly, it won't register as a click.
These values were chosen by what arbitrarily "feels good" to me.